messaging/WFLY-14133_configurable_bridge_call_timeout.adoc

```diff
@@ -0,0 +1,150 @@
+= Add call-timeout attribute to the Artemis core bridge resource
+:author:            Tomas Hofman
+:email:             thofman@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+The Artemis broker defines a call-timeout attribute on the core bridge, which sets a time out for blocking calls to the
+server. Unless modified, the call-timeout defaults to 30 seconds (this is the default value defined in Artemis).
+It is possible to programmatically modify this timeout by a lower level Artemis API, but it is not currently possible
+to modify with the high level API that Wildfly currently uses to create core bridges. It is currently not possible at all to
+modify this attribute by an EAP user / operator.
+
+It was requested by GSS to make this attribute configurable via the Wildfly/JBoss EAP management model.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/EAP7-1622
+* https://issues.redhat.com/browse/WFLY-14133
+
+=== Related Issues
+
+* https://issues.apache.org/jira/browse/ARTEMIS-3074
+* https://issues.redhat.com/browse/JBEAP-20326
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:mkopecky@redhat.com[Marek Kopecky]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
```

Comment:
This RFE is tested by engineering, see the "Testing By" flag on EAP7-1622 jira.

Reply:
OK, fixed.